FirebaseCore/Sources/FIRHeartbeatLogger.m

```diff
@@ -65,8 +65,7 @@ - (FIRHeartbeatsPayload *)flushHeartbeatsIntoPayload {
   return payload;
 }
 
-// TODO: Rename to `heartbeatCodeForToday` in future PR's.
-- (FIRHeartbeatInfoCode)heartbeatCode {
+- (FIRHeartbeatInfoCode)heartbeatCodeForToday {
```

Comment:
I'm sure this has come up before, but I'd like to better understand `Today`.

- What if we decide we want more frequent heartbeats in the future? Or is the 24 hour window an ongoing constraint?
- Is today specific to the device, and how does it relate to timezone differences?

Reply:
https://github.com/firebase/firebase-ios-sdk/blob/4e7a6441a54076423b215d1fbb2f0e4fd338b20b/FirebaseCore/Extension/FIRHeartbeatLogger.h#L49-L55

This API is a direct replacement of `+[FIRHeartbeatInfo heartbeatCodeForTag:]` that should be used by product SDKs whose backend can only parse single a single day's heartbeat code (`FIRHeartbeatInfoCode`).

---

> What if we decide we want more frequent heartbeats in the future?

This is a one line change in the heartbeats logging source that would be reflected in the return type of this API:
https://github.com/firebase/firebase-ios-sdk/blob/4e7a6441a54076423b215d1fbb2f0e4fd338b20b/FirebaseCore/Extension/FIRHeartbeatLogger.h#L41-L47

The `heartbeatCodeForToday` is never designed to send more than one heartbeat. As more backends migrate to parse payloads with more heartbeats, then this `heartbeatCodeForToday` would be removed entirely.
---

> Or is the 24 hour window an ongoing constraint?

It is an ongoing constraint for product SDKs that havenot upgraded their backends to parse requests with multiple heartbeats.

> Is today specific to the device, and how does it relate to timezone differences?

---

_Today_ is specific to device and is standardized to use the GMT time zone with no time offset.